scrapy/spiders/feed.py

```diff
@@ -31,7 +31,7 @@ def process_results(self, response, results):
         processing required before returning the results to the framework core,
         for example setting the item GUIDs. It receives a list of results and
         the response which originated that results. It must return a list of
-        results (Items or Requests).
+        results (Requests, Items, dicts, dataclasses).
```

Comment:
:lipstick:  We could alternatively modify the string not to refer to classes, but concepts.
```suggestion
        results (items or requests).
```